utils/clusterTester/columnstoreClusterTester.sh

```diff
@@ -728,12 +730,12 @@ checkPorts()
   pass=true
   for ipadd in "${NODE_IPADDRESS[@]}"; do
 
-    `sudo nmap $ipadd -p 8600-8630,8700,8800,3306 | grep 'filtered' > /tmp/port_test`
+    `nmap $ipadd -p 8600-8630,8700,8800,3306 | grep 'filtered' > ${tmpDir}/port_test`
```

Comment:
fwiw, my ubuntu CS installations don't have nmap; there should be an existence check first